src/Core/Addon/Theme/ThemeManager.php

```diff
@@ -484,22 +485,32 @@ public function saveTheme($theme)
         );
     }
 
+    /**
+     * @return \AppKernel|\Symfony\Component\HttpKernel\KernelInterface|void
+     */
+    private function getKernel()
+    {
+        global $kernel; // sf kernel
+
+        if (!(null !== $kernel && $kernel instanceof \Symfony\Component\HttpKernel\KernelInterface)) {
+            return;
```

Comment:
```suggestion
            return null;
```

return explicit null because of `if (null === $this->getKernel())`